### PR TITLE
Typo Fixes(PlusMigrationGuide.md)

### PR DESCRIPTION
**Pull request by Luro223**

### CODE REVIEW

**Comment by Luro223** [comment]
Need additional info related to "snapshot management" @isaak654

File: Content/PlusMigrationGuide.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The feature to copy an existing box can be found now on another place. (See the
 
 ![](../Media/Migration/3.png)
 
-To copy a existing box configuration the "Duplicate Sandbox" menu comand can be used.
+To copy a existing box configuration the "Duplicate Sandbox" menu command can be used.
```
Comment (isaak654) [inline Content/PlusMigrationGuide.md]:
```suggestion
To copy a existing box configuration, the "Duplicate Sandbox" menu command can be used.
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Comment by isaak654** [comment]
> Need additional info related to "snapshot management" @isaak654

I doubt it, this is a generic comparison between the features of Sandboxie Classic (which has no snapshot management) and Plus ones.

File: Content/PlusMigrationGuide.md [hunk 2]
```diff
@@ -72,61 +68,61 @@ Sandboxie-Plus can not only warn when unboxed processes are started, but it can
 
 ![](../Media/Migration/10.png)
 
-On this tab the shell integration can be configured, most functions are available, although depecated windows features were dropped and other options were moved out.
+On this tab, the shell integration can be configured. Most functions are available, although deprecated features were dropped and other options were moved out.
```
Comment (isaak654) [inline Content/PlusMigrationGuide.md]:
```suggestion
On this tab, the shell integration can be configured. Most functions are available, although some deprecated features were dropped and other options were moved out.
```

File: Content/PlusMigrationGuide.md [hunk 3]
```diff
@@ -138,31 +134,31 @@ The new UI supports groups just like the old one.
...
 
-Sandboxie plus can not only use the old method of blocking internet access but also use the Windows Filtering Platform (WFP) instead, which provides better compatibility.
+Sandboxie Plus can not only use the old method of blocking internet access but also the Windows Filtering Platform (WFP), which provides better compatibility.
 
 ## Network Access
 
 ![](../Media/Migration/25.png)
 
-Aditionally using the WFP facility a full per box firewall can be configured (tab 'Internet Restrictions' --> sub tab 'Network Firewall Rules').
+Additionally, using the WFP facility, a full per sandbox firewall can be configured (tab "Internet Restrictions" --> sub tab "Network Firewall Rules").
```
Comment (isaak654) [inline Content/PlusMigrationGuide.md]:
```suggestion
Additionally, using the WFP facility, a per sandbox firewall can be configured (tab "Internet Restrictions" --> sub tab "Network Firewall Rules").
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: Content/PlusMigrationGuide.md [hunk 4]
```diff
@@ -1,36 +1,32 @@
...
-
+This guide shows where all the known Sandboxie functions can be found in the new UI.
 
 ## Main Window
 
 ![](../Media/Migration/0.png)
 
-The over all layout of the main window of sandman.exe is exactly the same as the old one of sbiectrl.exe when the "Simple View" is chosen. If the "Advanced View" is chosen there are three additional tabs on the bottom of the window ("Sbie Messages" etc.), so it corresponds with what can be seen in the right picture.
-
+The over all layout of the main window of sandman.exe is exactly the same as the old one of sbiectrl.exe when the "Simple View" is chosen. If the "Advanced View" is chosen, there are three additional tabs on the bottom of the window ("Sbie Messages" etc.), so it corresponds with what can be seen in the right picture.
```
Comment (isaak654) [inline Content/PlusMigrationGuide.md]:
```suggestion
The overall layout of the main window of SandMan.exe is exactly the same as the old one in Sbiectrl.exe when the "Simple View" is chosen. If the "Advanced View" is chosen, there are three additional tabs on the bottom of the window ("Sbie Messages" etc.), so it corresponds with what can be seen in the right picture.
```

File: Content/PlusMigrationGuide.md [hunk 5]
```diff
@@ -1,60 +1,56 @@
...
 
-The modern Sandboxie UI has replaced by the "Files and Folders" view to a separate window which can be opened form the box context menu.
+The modern Sandboxie UI has replaced the "Files and Folders" view with a separate window that can be opened from the box context menu.
 
 ## Files and Folders - view / window
 
 ![](../Media/Migration/6.png)
 
-The window 'Files' offers the same functionality as the old view, but enhances it by providing a full windows context menu.
+The window "Files" offers the same functionality as the old view, but enhances it by providing a full windows context menu.
```
Comment (isaak654) [inline Content/PlusMigrationGuide.md]:
```suggestion
The window "Files" offers the same functionality as the old view, but enhances it by providing a full context menu.
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Review by isaak654 — APPROVED** [review]

**Pull request merged**